source/plot/SvgWriter2D.ooc

```diff
@@ -10,11 +10,11 @@ import LinePlotData2D
 import svg/Shapes
 
 SvgWriter2D: class {
+	numberOfPlotsHorizontally: Int
```

Comment:
Hmm we should never have public members in classes. This must have slipped through. Can you fix this now (as this PR is pretty small anyway)